docs/usage/getting-started/private-packages.md

```diff
@@ -265,6 +265,18 @@ Renovate doesn't support reading `npmRegistries` and `npmScopes` from `.yarnrc.y
 Renovate updates `npmRegistries` in `.yarnrc.yml` with resolved `hostRules` before running Yarn.
 For Renovate to overwrite existing `npmRegistries` entry, the key should match the `matchHost` without the protocol (`http:` or `https:`) and with the trailing slash.
 
+For example, Renovate configuration above this section will update `.yarnrc.yml` as following:
+
+```yaml
+npmRegistries:
+  //npm.pkg.github.com/:
+    npmAuthToken: <Decrypted PAT Token>
+  //npm.pkg.github.com:
+    # this will not be overwritten
+  https://npm.pkg.github.com/:
+    # this will not be overwritten
```

Comment:
My intention was as they are valid Yarn configuration, they would conflict with the configuration set by Renovate, so the last sentence was to clear them.